flake.nix

```diff
@@ -3,7 +3,7 @@
 
     inputs.haskellNix.url = "github:input-output-hk/haskell.nix";
     inputs.nixpkgs.follows = "haskellNix/nixpkgs-unstable";
-    inputs.flake-utils.url = "github:hamishmack/flake-utils/hkm/nested-hydraJobs";
```

Comment:
What was doing the `hkm/nested-hydraJobs` thing?

Reply:
That had patches that were eventually upstreamed.